test/Microsoft.AspNet.Mvc.Core.Test/Controllers/DefaultControllerActivatorTest.cs

```diff
@@ -46,16 +98,23 @@ public void Create_TypeActivatesTypesWithServices()
             serviceProvider.Setup(s => s.GetService(typeof(TestService)))
```

Comment:
I think i've done this, I don't know why github still shows it, still is not part of my changes :)